build.sbt

```diff
@@ -21,19 +21,19 @@ lazy val commonSettings =
 lazy val emailaddress = Project(appName, file("."))
   .settings(publish / skip := true)
   .aggregate(
-    play28,
+    //play28,
```

Comment:
I am the worst offender for doing this, but don’t  leave commented code in when you commit. :)
Thanks.

Reply:
Yeah definitely, I will remove this comment.